snap/hooks/post-refresh

```diff
@@ -33,8 +33,12 @@ should_transition_to_allowed_users() {
 
 transition_to_allowed_users() {
   log "Transitioning to allowed users"
-  install -D --target-directory "${SNAP_DATA}/broker.conf.d" --mode=0700  \
-    "${SNAP}/conf/migrations/pre-${INITIAL_ALLOWED_USERS_VERSION}/broker.conf.d/"*
+  src_dir="${SNAP}/conf/migrations/pre-${INITIAL_ALLOWED_USERS_VERSION}/broker.conf.d"
+  dest_dir="${SNAP_DATA}/broker.conf.d"
+  # shellcheck disable=SC2174 # it's fine that --mode only applies to the deepest directory, because the SNAP_DATA
+  # directory is created by snapd with the correct permissions.
+  mkdir -p --mode=0700 "${dest_dir}"
+  cp --update=none "${src_dir}/"* "${dest_dir}"
```

Comment:
are you sure this would copy those files with the desired mode?

Reply:
fixed